### DISCUSSION TITLE
Option to disable the @ short-hand

### DISCUSSION THREAD

**Discussion by Drammy** [alpinejs/alpine#3300] [3. Feature ideas]
### Discussed in https://github.com/alpinejs/alpine/discussions/3207

<div type='discussions-op-text'>

<sup>Originally posted by **Drammy** October 12, 2022</sup>
I would like to use alpine alongside .net core Blazor WASM which uses the @ sign to markup c# code.

Is it possible to disable the short-hand @ usage in alpine so that there is no clash between two technologies?</div>

**Comment by joshhanley** [comment]
This issue has been converted to a discussion as we are now using discussions to track bugs and ideas.
See this announcement for more details https://github.com/alpinejs/alpine/discussions/1423

**Comment by Drammy** [comment]
Since updating to alpine 3.12 I'm getting an exception (Safari, not tested other browsers) when removing the @ shorthand, like so..
![image](https://user-images.githubusercontent.com/5029037/224317127-07ed736f-ca91-4b8e-b41b-6f3d19588ea7.png)

Here's the exception
`TypeError: attr.value.startsWith is not a function. (In 'attr.value.startsWith('@')', 'attr.value.startsWith' is undefined)`
![image](https://user-images.githubusercontent.com/5029037/224317708-9b0181f7-791c-4b6d-80fd-2b1f110392e8.png)

Is there a new way of removing alpine's @ shorthand so it doesn't conflict with Blazor?

**Reply by ekwoka** [reply]
That's curious. It doesn't seem like any of the merged PRs should impact this.

We might just want to give something more specific for removing this.

**Reply by Drammy** [reply]
Just checked - we updated from 3.10.5.
Have downgraded in the meantime :)
